lib/Dialect/FIRRTL/Transforms/GrandCentralTaps.cpp

```diff
@@ -273,6 +273,73 @@ static FailureOr<Literal> parseIntegerLiteral(MLIRContext *context,
   return lit;
 }
 
+Value lowerInternalPathAnno(AnnoPathValue &srcTarget,
+                            const AnnoPathValue &moduleTarget,
+                            const AnnoPathValue &target,
+                            StringAttr internalPathAttr, FIRRTLType targetType,
+                            ApplyState &state) {
+  Value sendVal;
+  FModuleLike mod = cast<FModuleLike>(moduleTarget.ref.getOp());
+  InstanceOp modInstance;
+  if (!moduleTarget.instances.empty()) {
+    modInstance = moduleTarget.instances.back();
+  } else {
```

Comment:
Is this going to do the right thing if there is an instance path with size 2 or greater?

Reply:
Here, the idea is that if the user specified a non local path, then the `moduleTarget.instances` will be non empty, and then the last instance in the array should be the instance for `mod`. Otherwise, we query the `InstanceGraph` to get the unique instance for `mod`.